media-sound/jack/jack-3.1.1-r2.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+PYTHON_REQ_USE="ncurses"
+inherit python-single-r1
```

Comment:
Why are you using python-single-r1 instead of distutils-r1? You are patching setup.py but not using it, and reinventing the whole install.